💄 Add base figma colors

matheusvellone opened the pull request:
Got base colors from ellipse upper row options and added to the widget.
I choose not to add the bottom ones because the options would not be aligned like it is in the ellipse.

Also used a function from [this](https://stackoverflow.com/questions/3942878/how-to-decide-font-color-in-white-or-black-depending-on-background-color) StackOverflow question do dynamically choose White/Black color for Table name based on the background color, to have always a better contrast

![image](https://user-images.githubusercontent.com/5989971/168114150-2ea96c51-1de2-4e9b-b1fc-e138db632ccc.png)
![image](https://user-images.githubusercontent.com/5989971/168116245-38e202f5-57de-4f07-ac1d-5ef6a7c59b06.png)
![image](https://user-images.githubusercontent.com/5989971/168116299-5428c988-d3bb-4f1f-8ad1-177c24f0ef84.png)

matheusvellone committed:
💄 Add base figma colors

PillFall reviewed (changes requested):
Great, love it, I was thinking in just that. The colors I use in the Widget are pretty arbitrary (those were the colors I was using for my project), and I just totally forgot to change those.

There are only three things I want you to check before merging this PR.

- I know I still do not have contributing guide and code / commit styling, but I want the commits to have an according [gitmoji](https://gitmoji.dev/), and start capitalized, without end dot. (Required)
- The Figma colors are documented, and you can get those from UI Figma project, I really want to adhere to those
  - `#1e1e1e` Black
  - `#b3b3b3` Gray
  - `#f24822` Red
  - `#ffa629` Orange
  - `#ffcd29` Yellow
  - `#14ae5c` Green
  - `#0d99ff` Blue
  - `#9747ff` Violet
  - `#ffffff` White
- I believe that the calculation of the `chooseTextColor` function can be simplified using the `Color` interface, but at this point, I wouldn't be necessary.
```typescript
// r, b, g, a are numbers from 0 to 1 where {0,0,0,1} is black, and {1,1,1,1} is white.
interface Color {
  r: number,
  g: number,
  b: number,
  a: number
}
```

matheusvellone commented:
>The Figma colors are documented

Hmm, didn't know that. Got the colors by using a color picker. Changed the colors to the list you sent ✅ 

Also fixed commit style and fixed return type for `chooseTextColor` ✅

PillFall reviewed (changes requested):
Cool, but there is still a missing thing, the first word in the subject must be capitalized

PillFall reviewed (approved):
Ok, is all perfect, I'll merge later this day
